kroxylicious/src/main/resources/META-INF/deptective.json

```diff
@@ -0,0 +1,48 @@
+{
+  "components" : [ {
+    "name" : "io.kroxylicious.proxy",
+    "contains" : [ "io.kroxylicious.proxy" ],
+    "reads" : [ "io.kroxylicious.proxy.filter", "io.kroxylicious.proxy.internal", "io.kroxylicious.proxy.internal.filter", "io.netty.*", "org.apache.logging.log4j" ]
```

Comment:
To confirm I understand this:

* `name` is a logical name for this "rule"?
* `contains` is a package name or pattern for matching packages?
* `reads` is basically "is allowed to import/use from these other packages"? And may be package names or patterns